reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
Hi @ticoann 
No, I don't use the setenvscripts.sh, it's pointing to old script location. We may need to update it.
My procedure for running a test is the following:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/CompOpsWorkflowOperationsWMAgentScripts#Setting_Up_Scripts
Basically on vocms049 (or any other agent-machine):
voms-proxy-init -voms cms
export X509_USER_PROXY=/tmp/x509up_uXXXX
source /data/srv/wmagent/current/apps/wmagent/etc/profile.d/init.sh
I don't ask for any specific python version, specifically in vocms049, points to this python:
/data/srv/wmagent/v1.0.8.patch1/sw.pre/slc6_amd64_gcc481/external/python/2.6.8-comp9/bin/python